en/docs/get-started/quickstart.md

```diff
@@ -0,0 +1,148 @@
+    # WebSubHub consolidator endpoint (eg: http://localhost:10001)
+    url = "<websubhub-consolidator-state-snapshot-endpoint>"
+```
+
+Update the configurations related to broker connection in the `conf/Config.toml`.
+
+```toml
+    [websubhub.config.kafka.connection]
+    bootstrapServers = "<kafka-bootstrap-server-url>"
+```
+
+## Step 4: Start WSO2 Integrator: WebSubHub
+
+Start the WSO2 WebSubHub Consolidator.
+
+```sh
+    $ cd wso2websubhub-consolidator-1.0.0
+    $ ./bin/wso2websubhub-consolidator.sh
+```
+
+Start the WSO2 WebSubHub.
+
+```sh
+    $ cd wso2websubhub-1.0.0
+    $ ./bin/wso2server.sh
+```
+
+## Step 5: Invoke publisher operations
+
+Register a topic in the WSO2 WebSubHub.
+
+```sh
+    $ curl -X POST https://localhost:<websubhub-port>/hub \
+        -H "Content-Type: application/x-www-form-urlencoded" \
+        -d "hub.mode=register" \
+        -d "hub.topic=<topic-name>" \
+        -k
+```
+
+Publish content to the WSO2 WebSubHub.
+
+```sh
+    $ curl -X POST "https://localhost:<websubhub-port>/hub?hub.mode=publish&hub.topic=<topic-name>" \
+        -H "Content-Type: application/json" \
+        -d '{
+            "message": "This is a test message"
+        }' \
+        -k
+```
+
+## Step 6: Register a subscriber
+
+Go to <a href = "https://webhook.site/">https://webhook.site/</a> and create a new webhook with the following configurations.
+
+<a href="{{base_path}}/assets/img/get-started/quickstart/webhook-subscriber-configure.png"><img src="{{base_path}}/assets/img/get-started/quickstart/webhook-subscriber-configure.png" alt="Configure webhook subscriber" width="60%" style="padding-top: 20px" ></a>
+
+Copy the unique URL.
+
+<a href="{{base_path}}/assets/img/get-started/quickstart/webhook-subscriber-url.png"><img src="{{base_path}}/assets/img/get-started/quickstart/webhook-subscriber-url.png" alt="Copy unique URL" width="60%" style="padding-top: 20px" ></a>
+
+URL-encode the copied unique URL and refer to it as **encoded-callback-url**.
+
+Initiate the subscription call using the following cURL.
+
+```sh
+    $ curl -X POST "https://localhost:<websubhub-port>/hub" \
+        -H "Content-Type: application/x-www-form-urlencoded" \
+        -d "hub.topic=<topic-name>" \
+        -d "hub.callback=<encoded-callback-url>" \
+        -d "hub.mode=subscribe" \
+        -d "hub.secret=<subscriber-secret>" \
+        -d "hub.lease_seconds=50000000" \
+        -k
+```
+
+Publish the content to the relevant topic in WSO2 WebSubHub, and you can view the delivered content in the configured webhook endpoint.
+
+<a href="{{base_path}}/assets/img/get-started/quickstart/webhook-subscriber-content-delivery.png"><img src="{{base_path}}/assets/img/get-started/quickstart/webhook-subscriber-content-delivery.png" alt="Copy unique URL" width="60%" style="padding-top: 20px" ></a>
+
+## Congratulations
+
+You have successfully completed **WSO2 Integrator: WebSubHub** quickstart. 
+
+To learn more about it, we suggest following the next steps:
```

Comment:
Next step should refer to the next page coming in the nagivation. In the bottom navibation navigation next section points to `Connecting to an external Idp` so it is not matching.